.github/workflows/v2-nightly.yml

```diff
@@ -1,53 +1,54 @@
+name: v2 Nightly Tests
 on:
   workflow_dispatch: # manually
   schedule:
     - cron: "0 0 * * *"
 
 jobs:
   appsec-v2:
-    uses: ./.github/workflows/appsec.yml
+    uses: DataDog/dd-trace-go/.github/workflows/appsec.yml@v2-dev
```

Comment:
That makes them tremendously painful to update (since it needs to be on the branch before it can be tested); but unfortunately GitHub offers no better way...

Reply:
I'm not fond of this too, but you said it: GitHub offers no better way. Thanks for the approval!